adage/analyze/tests.py

```diff
@@ -432,25 +448,25 @@ def call_get_API(self, uri):
         '''
         Helper method: asserts that GET method is allowed via input URI.
         '''
-        resp = self.api_client.get(uri, data={'format': 'json'})
```

Comment:
Just checking - removing ``data={'format': 'json'}`` here is not because of the glitch right? It just isn't necessary when making test requests with the test's 'api_client'?